### ISSUE TITLE
Use of special characters like "æ, ø, å" crashes app with no error

### ISSUE THREAD

**Issue by Blazeflack** [justinknguyen/BeatBind#17]
**Describe the bug**
If you use any special characters for hotkeys, such as the Scandinavian "æ", "ø" and "å", then the app will not start when pressing the "Start & Close" button. It fails silently with no shown error, and I was not able to find any kind of logfile. Replacing the characters with non-special characters made the program start up succesfully.

**To Reproduce**
1. Set any hotkey to use any of the special characters I mentioned.
2. Click on "Save" button.
3. Click on "Start & Close" button.
4. Notice app fails to start.

**Expected behavior**
I expected the app to start.

**Screenshots**
Not applicable.

**Desktop (please complete the following information):**
 - OS: Windows 11 23H2, build 22631.3447
 - Version 1.5.2 (I also checked with version 1.5.1)

**Additional context**
Here is the hotkey settings taken from beatbind-config.json:
```json
    "hotkeys": {
        "play/pause": "control+\u00e6",
        "prev_track": "control+alt+\u00e6",
        "next_track": "control+\u00f8",
        "volume_up": "control+shift+\u00f8",
        "volume_down": "control+shift+\u00e6",
        "mute": "",
        "seek_forward": "",
        "seek_backward": ""
    }
```

**Comment by justinknguyen** [comment]
This app uses the global_hotkeys module by btsdev: https://github.com/btsdev/global_hotkeys. I recommend creating this issue ticket there so they can solve your issue.

It has a list of supported keys at the bottom of its README. You can ask if they can support special keys like that

**Comment by justinknguyen** [comment]
give me a ping here to let me know once your issue is fixed, then I'll update the app. Thanks!

**Comment by Blazeflack** [comment]
I have requested that support be added for these characters: https://github.com/btsdev/global_hotkeys/issues/9

Awaiting the results.

**Comment by justinknguyen** [comment]
> I have requested that support be added for these characters: https://github.com/btsdev/global_hotkeys/issues/9
> 
> 
> 
> Awaiting the results.

just checked out his answer and I guess I'll have to look into it further. Is it just those 3 characters that you want?

**Comment by justinknguyen** [comment]
@Blazeflack I added those characters in [v1.5.3](https://github.com/justinknguyen/BeatBind/releases/tag/v1.5.3), but I'm not too sure if it actually works.

There was a bug after I recently updated the `global_hotkeys` module where the hotkey input field listener wasn't working in v1.5.2, causing the ability to enter those special characters. When I switched to the Norwegian keyboard, the hotkey input field listener detects `æ` as ', `ø` as \`, and `å` as ]. However, the hotkey was able to trigger when I pressed `æ` outside of the app.

So even if the hotkey input field listener displays the incorrect character for you, it should still work. Let me know if it doesn't.
